packages/forklift-console-plugin/src/modules/NetworkMaps/NetworkMappingsPage.tsx

```diff
@@ -85,29 +81,29 @@ const Page = ({
   const isLoading = !isLoadSuccess && !isLoadError;
   const loadedDataIsEmpty = isLoadSuccess && !isLoadError && (data?.length ?? 0) === 0;
 
+  if (isLoading) {
+    return <Loading />;
+  }
```

Comment:
This isn't enough as the spinner will show any time a load or a reload or a refresh is in progress (as determined by the hook itself).

The base `isLoading` probably needs to be more like "if (isInitialLoad) {".  The table can handle a loading spinner itself past the initial load.

Reply:
that is what @rszwajko is working on in :https://github.com/kubev2v/forklift-console-plugin/pull/312